memory/src/main/java/org/opensearch/ml/memory/action/conversation/UpdateConversationRequest.java

```diff
@@ -35,7 +36,7 @@ public class UpdateConversationRequest extends ActionRequest {
     private String conversationId;
     private Map<String, Object> updateContent;
 
-    private static final Set<String> allowedList = new HashSet<>(Arrays.asList(META_NAME_FIELD));
+    private static final Set<String> allowedList = new HashSet<>(Arrays.asList(META_NAME_FIELD, META_ADDITIONAL_INFO_FIELD));
```

Comment:
This would allow update/delete anything inside META_ADDITIONAL_INFO_FIELD. I think you need to put some restrictions to not allow updating certain fields inside it like the application/memory type, etc?

Reply:
Yes, that makes sense. We can initially make it open and then add restrictions when the need arises.